source/jsonnet/england-wales/individual/blocks/employment/employment_status_last_seven_days.jsonnet

```diff
@@ -73,11 +79,11 @@ local proxyAnswerDescription = 'Freelance means that they are self-employed and
   page_title: 'Employment status in the last seven days',
   question_variants: [
     {
-      question: question(nonProxyTitle, nonProxyAnswerDescription),
+      question: question(nonProxyTitle, nonProxyQuestionDescription, nonProxyAnswerDescription),
       when: [rules.isNotProxy],
     },
     {
-      question: question(proxyTitle, proxyAnswerDescription),
+      question: question(proxyTitle, proxyQuestionDescription, proxyAnswerDescription),
```

Comment:
If we're making changes we should adhere to the style guide and use a single variable for proxy (isProxy) and functions for the things that vary.